src/main/resources/db/changelog/changes/v1.1.0/update-drv-purchase-order-line-details-definition.xml

```diff
@@ -262,26 +262,27 @@
                 "dataType": "stringType"
               },
               "valueGetter": "purchase_order.jsonb ->> 'orderType'",
+              "filterValueGetter": "\"left\"(lower(${tenant_id}_mod_orders_storage.f_unaccent(purchase_order.jsonb ->> 'orderType'::text)), 600)",
```

Comment:
We should add valueFunctions everywhere where we have filterValueGetters. For example, right here, we can have:

"valueFunction": "\"left\"(lower(${tenant_id}_mod_orders_storage.f_unaccent(:value)), 600)",